domains.txt

```diff
@@ -649,16 +724,20 @@ streamrecruitment.co.uk
 strikeits.com
 surestart-recruitment.co.uk
 swanitrecruitment.com
+synergizecl.co.uk
 synergy-emea.co.uk
 syntaxconsultancy.com
 t2cuk.com
 t4ssolutions.co.uk
+tab-it-recruitment.co.uk
+tagmatix360.com
 tailormack.com
 talentcluster.co.uk
 talentinternational.co.uk
 talentinternational.com
 talentpoint.co
 talentspa.co.uk
+talenus.co.uk
```

Comment:
Talenus is a specialist recruitment consultancy focusing on matching industry leading talent with some of the world’s most innovative companies within the UK & Europe.